android/src/toga_android/widgets/base.py

```diff
@@ -98,11 +100,6 @@ def set_bounds(self, x, y, width, height):
             self.container.set_child_bounds(self, x, y, width, height)
 
     def set_hidden(self, hidden):
-        view = View(self._native_activity)
-        if not view.getClass().isInstance(self.native):
-            # save guard for Widgets like Canvas that are not based on View
-            self.interface.factory.not_implemented("Widget.set_hidden()")
-            return
```

Comment:
Not sure why this was done as an isinstance check, rather than by overriding in the subclass, but I've moved it.